ci-operator/step-registry/edge-nodes/machineset-manifests/outpost/edge-nodes-machineset-manifests-outpost-commands.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+REGION="${LEASED_RESOURCE}"
+export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+
+
+MACHINE_ROLE="worker"
+if [[ ${EDGE_NODE_WORKER_ASSIGN_PUBLIC_IP} == "yes" ]]; then
+  subnet_id=$(head -n 1 "${SHARED_DIR}/outpost_public_id")
+else
+  subnet_id=$(head -n 1 "${SHARED_DIR}/outpost_private_id")
+fi
+
+zone_name=$(yq-go r "${SHARED_DIR}/availability_zones" '[0]')
+machineset_name_postfix=${RANDOM:0:2}
+
+# keeping manifest_ prefix as this step can be used in manifest injection before installation 
+edge_zone_machineset="${SHARED_DIR}/manifest_edge_zone_machineset.yaml"
```

Comment:
shall we update `edge_zone` to `edge_node` accordingly?

Reply:
Updated.